org.hl7.fhir.r5/src/main/java/org/hl7/fhir/r5/utils/ToolingExtensions.java

```diff
@@ -802,6 +811,20 @@ public static String getLanguageTranslation(Element element, String lang) {
     return null;
   }
 
+  public static StringType getLanguageTranslationElement(Element element, String lang) {
+    for (Extension e : element.getExtension()) {
+      if (e.getUrl().equals(EXT_TRANSLATION)) {
+        Extension e1 = ExtensionHelper.getExtension(e, "lang");
+
+        if (e1 != null && e1.getValue() != null && e1.getValue() instanceof CodeType && ((CodeType) e1.getValue()).getValue().equals(lang)) {
```

Comment:
Are we only accepting exact matches for lang? If for example, the extension has `en` as the lang, it looks to me like this will not even attempt to translate `en_US`

Reply:
Good point, and one that's on my todo list

Reply:
I think there's some logic here that may be of use: https://devdocs.io/openjdk~17/java.base/java/util/locale#filter(java.util.List,java.util.Collection)